micropub.php

```diff
@@ -197,6 +235,25 @@ private static function authorize() {
     return NULL;
   }
 
+  private static function return_query($user_id) {
+    switch($_GET['q']) {
+      case 'syndicate-to':
+      // Fallback
+      case 'mp-syndicate-to':
+        status_header(200);
```

Comment:
I get the first line is the header line, what is the second? The 200 return wouldn't happen with an unsupported query string.